### DISCUSSION TITLE
Changing Runtime Value during WhileLoop

### DISCUSSION THREAD

**Discussion by ypo777** [testsigmahq/testsigma#180] [Questions & Help (How to)]
I have a test case which is to search element from table and get rowNumber of that element.
First I stored initial row=1 as runtime value. [ ELEMENT = //tr[$|row|//td[7]  ]  <br>
WhileLoop has a condition it will check **row is visible on the page** , then **IF Condition**  will check **row** has certian test data I want. If it has **BREAK LOOP**. 
**ELSE** row will increase to 2 using **Custom Function**.  <br>
The Problem -  increaseed **row** value  is not used in **WHILE LOOP** it is still using **row = 1**
It will become infinite loop.

**Comment by amitsavyasachi** [comment]
@ypo777 Looks like an bug, can you help me with a screenshot of execution?
